app/controllers/symptoms_controller.rb

```diff
@@ -0,0 +1,30 @@
+require 'fuzzystringmatch'
+
+class SymptomsController < ApplicationController
+  def search
+    if search_params[:symptom].empty? || search_params[:symptom] == ' ' || search_params[:when].empty?
+      flash[:error] = 'Please be sure to specify a symptom and when you experienced it'
+      redirect_to request.referer
+    else
+      jarow = FuzzyStringMatch::JaroWinkler.create(:native)
+      symptoms = current_user.medications.map do |med|
```

Comment:
this is where `Symptom.potential_symptoms(current_user)` would come in from PR #65

Reply:
Added 👍